etc/helm/pachyderm/templates/console/deployment.yaml

```diff
@@ -48,6 +48,10 @@ spec:
           value: {{ include "pachyderm.issuerURI" . | quote}}
         - name: REACT_APP_RUNTIME_ISSUER_URI
           value: {{ include "pachyderm.reactAppRuntimeIssuerURI" . | quote}}
+        {{- if .Values.oidc.avoidExternalNetwork }}
+        - name: DEX_BASE_URI
```

Comment:
How is DEX_BASE_URI different than ISSUER_URI?

Reply:
With this approach issuer uri is always the fqdn the user accesses (ingress.host). Dex_base will be basically straight to pachd